### ISSUE TITLE
Add Get URL ProtoFlux Nodes

### ISSUE THREAD

**Issue by lxw404** [Yellow-Dog-Man/Resonite-Issues#2487]
### Is your feature request related to a problem? Please describe.

Currently as far as I am aware, there is no way of getting a `URL` given any static asset reference.

### Describe the solution you'd like

There should be additional ProtoFlux nodes to take an asset and give the stored `URL`.

### Describe alternatives you've considered

The current best alternative is to extract the URL from the string representation of the asset itself, but I'm not sure if this is a reliable solution.

### Additional Context

One use case for this is to transfer the URL from an asset that a user provides to a tool, to one that the tool controls so that it can perform operations on a copy without affecting the original asset (such as image manipulation proxy calls etc.).

### Requesters

LuxKitty

**Comment by epicEaston197** [comment]
I don't think this would be something that would be added this is better suited for component access #57

**Comment by lxw404** [comment]
This has really no dependence on component access.  You can right now get an asset from an asset frame, however the properties are limited to those provided by current asset metadata components.  There are also asset protoflux nodes which allow you to get a limited set of properties as well, except for URLs.  This is just one missing property that isn't provided.
